packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -17,24 +17,29 @@ import { assertValidSchema, getKeyValPair } from "../lib/stateful";
 import type { Context, StackElem, State, StoreAccount, Txn } from "../types";
 
 export class Runtime {
-  store: State;
+  private store: State;
   ctx: Context;
 
   constructor (accounts: StoreAccount[]) {
     // runtime store
+    const assetInfo = new Map<number, AccountAssetInfo>();
     this.store = {
       accounts: new Map<string, StoreAccount>(),
-      globalApps: new Map<number, SSCParams>()
+      accountAssets: new Map<string, typeof assetInfo>(),
+      globalApps: new Map<number, SSCParams>(),
+      globalAssets: new Map<number, AssetParams>()
     };
+
+    // intialize accounts (should be done during runtime initialization)
+    this.initializeAccounts(accounts);
+
     // context for interpreter
     this.ctx = {
-      state: <State>{},
+      state: cloneDeep(this.store), // state is a deep copy of store
```

Comment:
I t hink @ratik21  already did this in his PR.

Reply:
Previously it was done in `executeTx` function only, but not at the time of construction. But for testing it properly, we need to have access without calling `executeTx`

Reply:
But how do you change `runtime.ctx` without calling `executeTx`? For which test do you need it?

Reply:
Here, only `store` is changed.
while testing opcodes, for ex: assetHolding,
assetHolding opcode require `ctx` but if we don't initialize it here, ctx will be blank as per previous code. One way will be remove `private` from store variable and initialize ctx in test.
In tests:
`interpreter.runtime.ctx.state = runtime.store`

Reply:
OK, let's keep what you wrote with `clone`.  Please add a comment to the code.

Reply:
ok